src/index.html

```diff
@@ -17,6 +17,88 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="logo_link"
+        href="index.html"
+      >
+        <img
+          class="logo_img"
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a
+              href="#"
+              class="is-active nav_link"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li
+            class="nav_item"
+            data-qa="hover"
+          >
+            <a
+              href="#"
+              class="nav_link"
+            >
+              Laptops & Computers
```

Comment:
Remember about correct indentation between parent and child elements. The `<a>` tag inside the `<li>` should be indented with 2 spaces.